src/rythm.js

```diff
@@ -13,6 +13,7 @@ export default class Rythm {
     this.addRythm('rythm-bass', 'pulse', 0, 10)
     this.addRythm('rythm-medium', 'pulse', 150, 40)
     this.addRythm('rythm-high', 'pulse', 400, 200)
+    this.animationFrameRequest = void 0
```

Comment:
What is the purpose of this line ?

Reply:
defaulting to `undefined` on load so we don't end up attempting to cancel an animation that doesn't exist ;)

Reply:
Animation cancel were already done in line 47, but i guess this is cleaner to cancel the animation request :+1:

Reply:
I completely forgot to save the request each frame - my bad - fixed ;)